website/docs/r/spring_cloud_accelerator.html.markdown

```diff
@@ -0,0 +1,69 @@
+---
+subcategory: "Spring Cloud"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_spring_cloud_accelerator"
+description: |-
+  Manages a Spring Cloud Accelerator.
+---
+
+# azurerm_spring_cloud_accelerator
+
+-> **NOTE:** This resource is applicable only for Spring Cloud Service with enterprise tier.
+
+Manages a Spring Cloud Accelerator.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example"
+  location = "West Europe"
+}
+
+resource "azurerm_spring_cloud_service" "example" {
+  name                = "example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku_name            = "E0"
+}
+
+resource "azurerm_spring_cloud_accelerator" "example" {
+  name                    = "default"
+  spring_cloud_service_id = azurerm_spring_cloud_service.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Spring Cloud Accelerator. Changing this forces a new Spring Cloud Accelerator to be created. The only possible value is `default`.
+
+* `spring_cloud_service_id` - (Required) The ID of the Spring Cloud Service. Changing this forces a new Spring Cloud Accelerator to be created.
+
+## Attributes Reference
```

Comment:
what does this resource do exactly? should there be more properties?

Reply:
This is a parent resource of customized accelerator resources.